src/containerapp/azext_containerapp/tests/latest/test_containerapp_create_update_with_source.py

```diff
@@ -0,0 +1,84 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import unittest
+
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, live_only)
+
+from azext_containerapp.tests.latest.utils import create_and_verify_containerapp_create_and_update, verify_containerapp_create_exception
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+class ContainerAppCreateTest(ScenarioTest):
+    @live_only()
```

Comment:
May I ask why do you mark these tests as `@live_only()`

Reply:
These tests require a `docker push` operation to push the image built as part of the test to the container registry and would not execute from the CI pipeline since docker is not installed in the CI.

Reply:
Updated.